packages/components/ng-ovh-payment-method/package.json

```diff
@@ -44,7 +44,6 @@
     "@ovh-ux/component-rollup-config": "^8.0.0 || ^9.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/manager-config": "^3.0.0 || ^4.0.0",
     "@ovh-ux/manager-core": "^11.0.0 || ^12.0.0",
```

Comment:
Anything planned for `@ovh-ux/manager-core` ?

Reply:
Planned :)